README.md

```diff
@@ -1,29 +1,106 @@
-# CNTK Object Recognition Video Tagging Tool
-
-This tool provides end to end support for generating datasets for and validating Object Recognition Models with CNTK.
-
-It supports the following scenarios:
-
-- Computer Assisted Tagging of Objects In Video Using Custom implementation of Camshift Algorithm
-- Export Tags to CNTK format for training a CNTK object recognition model.
-- Running and validating a trained CNTK object recognition model on new videos to generate stronger models. (Windows only for now, GPU recommended)
-
-## To Use:
- 1. Download the release binary.
- 2. Extract and run the app
- 3. Load a video.
- 4. Configure the tagging job specifying the following preferences
-    - Frame Extraction Rate (Number of frames to tag per a second)
-    - Tagging Region Type (Rectangle, Point, Square)
-    - Export Frames Until (How far to export)
-    - Labels (Labels for tagging)
- 5. Tag the video frame by frame.
- 6. Export Video to CNTK Format
- 7. Train model using [Object Detection using FastRCNN] (https://github.com/Microsoft/CNTK/wiki/Object-Detection-using-Fast-R-CNN#train-on-your-own-data).
- 8. Load new video apply model to new video, validate tags, re-export, retrain
- 9. Repeat step #8 on new videos until model preformance is satisfactory.  
-
-## Upcoming Features
+# CNTK Object Detection Video Tagging Tool
+
+This tool provides end to end support for generating datasets for and validating Object Detection Models with CNTK.
+
+**It supports the following scenarios:**
+
+- Computer assisted tagging and tracking of objects in video using the [Camshift tracking algorithm](http://opencv.jp/opencv-1.0.0_org/docs/papers/camshift.pdf) 
+- Exporting tags and assets to CNTK format for training a CNTK object detection model.
+- Running and validating a trained CNTK object detection model on new videos to generate stronger models. 
+
+## Table of Contents
+ - [Prerequisites](#prerequisites)
+ - [Tagging Job](#taggingjob)
+ - [Review/Iterate CNTK Object Detection Model](#review)
+ - [Upcoming Features](#upcoming)
+ - [How to Contribute](#contribute)
+ 
+-----------
+<a name="prerequisites"></a>
+## Prerequisites
```

Comment:
I would split this section into 2 parts so it will be clearer:
1. Installing and running the tagging tool
2. Installing CNTK with the FRCNN prequisites